internal/jobcontainers/process.go

```diff
@@ -0,0 +1,160 @@
+package jobcontainers
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os/exec"
+	"sync"
+
+	"github.com/Microsoft/hcsshim/internal/cow"
+	"github.com/Microsoft/hcsshim/internal/log"
+	"golang.org/x/sys/windows"
+)
+
+// JobProcess represents a process run in a job object.
+type JobProcess struct {
+	cmd            *exec.Cmd
+	stdioLock      sync.Mutex
+	stdin          io.WriteCloser
+	stdout         io.ReadCloser
+	stderr         io.ReadCloser
+	waitBlock      chan struct{}
+	closedWaitOnce sync.Once
+	waitError      error
+}
+
+var _ cow.Process = &JobProcess{}
+
+func newProcess(cmd *exec.Cmd) *JobProcess {
+	return &JobProcess{
+		cmd:       cmd,
+		waitBlock: make(chan struct{}),
+	}
+}
+
+// ResizeConsole - TODO (dcantah): Implement console support with new pseudo console api
+func (p *JobProcess) ResizeConsole(ctx context.Context, width, height uint16) error {
+	return nil
+}
```

Comment:
can we take this out for now?

Reply:
Won't compile as it won't implement the interface anymore. I suppose I could remove the compile time check for whether this implements the interface also but I'd rather leave this

Reply:
naw, in that case could you just return err.NotImplemented or something?

Reply:
Sure